vision/genesis_principles.md

```diff
@@ -0,0 +1,14 @@
+# ₿trust Genesis Principles
+₿trust’s Genesis Principles are an immutable set of principles governing its operation and outlining its vision. They exist to guide the board and representatives of the trust, and to act as an accountability safeguard against bad actors.  
+
+Board members will take ownership of this vision, and act in good faith to protect the interests and integrity of the organization. The specific implementation of these principles is the prerogative of the board and its selected representatives.
+
+1. Bitcoin will provide a monetary system that enhances financial sovereignty for all. ₿trust will support Bitcoin exclusively. In the event of a hard fork it will follow the chain endorsed by the user majority.
+
+2. ₿trust will maintain the highest levels of integrity. To that end, ₿trust does not exist to make a profit for itself or its representatives. The board will be mindful to prevent self-dealing within the organization and representatives will declare conflicts of interest as they arise.
+
+3. Free and open source projects are fundamental to Bitcoin’s censorship resistance and resilience. ₿trust will focus its efforts on fostering developer talent to work on under-resourced free and open source Bitcoin projects. It will prioritize projects based on how dependent the bitcoin ecosystem is on them, and their capacity to accept new contributors. 
```

Comment:
Been thinking that we might want to move `under-resourced` into the prioritizing sentence, because we might want to support projects that aren't nescisarily under-resourced?

Suggested copy: 
`₿trust will focus its efforts on fostering developer talent to work on free and open source Bitcoin projects. It will prioritize projects based on how dependent the bitcoin ecosystem is on them, resource requirements and their capacity to accept new contributors.`

Reply:
Good point! I think the suggestion is a better rewording to make the area of interest less restrictive. Additionally, we can chuck in the 'why' into your above suggestion, like so:

```suggestion
3. ₿trust will focus its efforts on fostering developer talent to work on free and open source Bitcoin projects, to maintain Bitcoin's censorship resistance and resilience. It will prioritize projects based on how dependent the bitcoin ecosystem is on them, resource requirements and their capacity to accept new contributors.
```

Reply:
This will break our pattern of starting with a "why" statement at the beginning of every principle, so I've left the "why" first here.